warehouse/ingest-core/src/main/java/datawave/ingest/data/config/ingest/IngestFieldFilter.java

```diff
@@ -1,42 +1,66 @@
 package datawave.ingest.data.config.ingest;
 
+import com.google.common.collect.Lists;
 import com.google.common.collect.Multimap;
-import com.google.common.collect.TreeMultimap;
 import datawave.ingest.data.Type;
-import org.apache.commons.lang.StringUtils;
+import datawave.ingest.data.config.GroupedNormalizedContentInterface;
+import datawave.ingest.data.config.NormalizedContentInterface;
+import org.apache.commons.collections.Unmodifiable;
+import org.apache.commons.collections.keyvalue.AbstractMapEntry;
+import org.apache.commons.lang3.StringUtils;
 import org.apache.hadoop.conf.Configuration;
 import org.apache.log4j.Logger;
 
+import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.Collection;
-import java.util.Map;
+import java.util.Collections;
+import java.util.List;
 
 /**
- * The field filter can drop fields based on the presence of other preferred fields. This is configuration driven and expects colon delimited pairs, A:B, where
+ * The field filter can drop fields based on the presence of other preferred fields.
+ * This is configuration driven and expects colon delimited pairs, A:B, where
  * if A is present in the fields, then any occurrences of B will be removed.
+ * In the case of the data.fieldvalue.filter, the values must match as well which
+ * implies that the number of KEEP fields must be the same as the number of DROP
+ * fields which is not the case for the fieldName filter.  For backward compatibility
```

Comment:
An explicit blurb about the & option would be nice.

Reply:
ok, will do